src/authdb.js

```diff
@@ -132,7 +168,9 @@ module.exports = {
   },
   async createUser (username, password, email) {
     const user = { username }
-    user.passwordHash = await bcrypt.hash(password, saltRounds)
+    if (password) {
+      user.passwordHash = await bcrypt.hash(password, saltRounds)
+    }
```

Comment:
This allows creating user without password, [existing code](https://github.com/immers-space/immers/blob/controlled-accounts/src/authdb.js#L64) in `validateUser` already checks and rejects authentication requests when the user has no saved password